app/components/windows/settings/Settings.vue.ts

```diff
@@ -181,7 +181,13 @@ export default class Settings extends Vue {
   }
 
   get categoryNames() {
-    return this.settingsService.getCategories();
+    return this.settingsService
+      .getCategories()
+      .filter(
+        category =>
+          !category.toLowerCase().startsWith('stream') ||
+          (category.toLowerCase().startsWith('stream') && category.length < 7),
```

Comment:
The stream settings for the second context need to be filtered out of this list. To make it less brittle, I opted to filter out any additional stream settings returned rather than the string `StreamSecond`.

Reply:
Yeah, that makes sense.  I think my only point was that if you're checking for a string of length less than 7 which starts with `stream`, the only thing it could be is `stream`, so you might as well make the check simpler.

Reply:
Ohhh gotcha. Changed